python/src/robyn/visualization/allocator_plotter.py

```diff
@@ -0,0 +1,328 @@
+from robyn.visualization.base_visualizer import BaseVisualizer
+from robyn.allocator.entities.allocation_results import AllocationResult
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from typing import Dict
+
+
+class AllocationPlotter(BaseVisualizer):
+    def __init__(self, result: AllocationResult):
+        super().__init__(style="bmh")
+        self.result = result
+        plt.style.use("bmh")
+        plt.rcParams.update(
+            {
+                "figure.figsize": (12, 8),
+                "axes.grid": True,
+                "axes.spines.top": False,
+                "axes.spines.right": False,
+            }
+        )
```

Comment:
looks like sizes are individually defined here within this component instead of base viz. I presume we will look at other common properties in next PR?